js/app.js

```diff
@@ -1,9 +1,61 @@
 // wait for DOM to load before running JS
-$(document).ready( function() {
+$(document).ready(function() {
 
   // check to make sure JS is loaded
   console.log('JS is loaded!');
 
   // your code here
 
-});
```

Comment:
Hey Bryan,

There are no extra points for speed so don't worry about how long it took you. I think you're doing fine and your code looks good.

Just a thought going forward though, I did notice that you had two loops in your data and I think you're doing extra work here. 

This code looks like it worked for you:
```
let searchResults = data.tracks;
           // first for loop to setup new array to loop through
           for(i=0; i < searchResults.items.length; i++){
             // variable for new array
           let newSong  = searchResults.items[i].name;
          let newArtist = searchResults.items[i].album.artists;
           // second for loop for new array
           for(j=0; j < newArtist.length; j++)

```
But saying searchResults = data.tracks and then newArtist = searchResults.items is doing the same work as just saying newArtists = data.track.items. So you could eliminate your first for loop through data.tracks and  and just iterate through data.tracks.items. Does that make sense?

Otherwise - great job. I'm really happy with your progress.

Matt